package.json

```diff
@@ -46,58 +46,58 @@
     "classnames": "2.2.5",
     "clean-webpack-plugin": "0.1.17",
     "compass-mixins": "0.12.10",
-    "copy-webpack-plugin": "4.1.1",
+    "copy-webpack-plugin": "4.2.0",
     "css-loader": "0.28.7",
     "dom4": "1.8.5",
     "es6-promise": "4.1.1",
-    "eslint": "4.8.0",
+    "eslint": "4.10.0",
     "eslint-plugin-lodash-fp": "2.1.3",
-    "eslint-plugin-promise": "3.5.0",
+    "eslint-plugin-promise": "3.6.0",
     "eslint-plugin-react": "7.4.0",
     "eslint-plugin-standard": "3.0.1",
     "file-loader": "1.1.5",
-    "i18next": "10.0.1",
+    "i18next": "10.0.7",
     "i18next-resource-store-loader": "0.1.2",
-    "js-cookie": "2.1.4",
+    "js-cookie": "2.2.0",
     "keymirror": "0.1.1",
     "lodash": "4.17.4",
     "markdown": "0.5.0",
     "moment": "2.19.1",
     "node-sass": "4.5.3",
+    "patch-package": "^3.6.0",
```

Comment:
Please remove the `^`.

Reply:
Read the README of this package. Looks like it is not used?

Reply:
Right. Uninstalled it but was left in the .json